### PR TITLE
Added option to not log 85°C readings. They appear if a DS18B20 has a power failure and returns this initial register value.

### PR DESCRIPTION
**Pull request by CC1337**
Hi, I am experiencing 85°C readings and unfortunately can't easily improve the sensor's power supply, so I added an option to ignore such readings (I would not treat it as "faulty" so I added another checkbox). Not sure how to limit this implementation to DS18B20 only. This PR contains a very easy solution as I only use DS18B20 currently.

Also I fixed a typo in 0.6.3's date :)

### CODE REVIEW

**Comment by Apollon77** [comment]
Hm ... The way you subscribe it it feels ver hacky. Somit really need to be done in the adapter? It more feels like you should do post processing as alias with a read function or with a small JavaScript which checks for these special invalid values and adjust the value as needed ...

**Comment by CC1337** [comment]
Well, actually this option did not come into my mind, I went for the adapter as I am directly saving those values via the InfluxDB adapter. But with one JS which is listening and writing to new instances could also be an option.
I've set up ioBroker 1.5 month ago so I am not yet familiar with all best practices. 
Doing that in the adapter would also solve the same "issue" for others that cannot optimize the wiring now anymore without need to reimplement the same state postprocessing. But if that's not what you are targeting, feel free to decline the PR.

**Comment by GermanBluefox** [comment]
It is no problem for this PR, but if people will say, that other values should be ignored too, so that should be done more generic.
E.g, "Ignore following values: 85, 92, -1"

**Comment by CC1337** [comment]
I'm not familiar with other onewire devices - and if there are use cases where default values could be different than -1 or 85. If you know there are, I definitely agree. It might make sense to configure that per sensor as well (as excluding 85°C only make sense for DS18B20). I also thought about some plausibility check (current value in a range of +/- x degrees -> 85°C is valid) but I am not sure if this (for me hopefully never reachable case unless something is burning) is actually worth it.

So we now have the "ignore errors" option, and "my" ignore 85°C option. The ignore error option basically excludes -1 values? I could add an input instead of the two checkboxes, but I have no clue how to do proper backwards compatibility when users update the adapter.
I would maybe rather leave the existing ignoring "-1" untouched if it's a more generic error and add another input for to be ignored values with some tooltip about the 85 / DS18B20 behavior as built-in documentation. What do you think?
I actually thought about it and then ... YAGNI :P

File: admin/index_m.html [hunk 1]
```diff
@@ -297,6 +297,10 @@
                     <input class="value" id="noStateChangeOnError" type="checkbox"/>
                     <label class="translate" for="noStateChangeOnError">No state change on error</label>
                 </div>
+                <div class="col s12 m4">
+                    <input class="value" id="noStateChangeOn85C" type="checkbox"/>
+                    <label class="translate" for="noStateChangeOn85C">No state change on 85°C reading (initial state of DS18B20 in case of power failure)</label>
```
[the remark this answers is not in the document]
Comment (CC1337) [reply admin/index_m.html]: Great tool, done!

File: main.js [hunk 2]
```diff
@@ -358,7 +358,11 @@ function readWire(wire) {
                             // else some float value, e.g. temperature
                             let val = owfsParseFloat(result.value);
                             if (!isNaN(val)) {
-                                adapter.setState('wires.' + wire._name, {val: val, ack: true, q: 0});
+                                if (adapter.config.noStateChangeOn85C && val == 85.0) {
```
[the remark this answers is not in the document]
Comment (CC1337) [reply main.js]: It was working fine since the above commit, but I do not mind changing it to prevent possible rounding errors that might be safer. Done!

**Comment by GermanBluefox** [comment]
Looks good :) Thank you.

**Pull request merged**